README.md

```diff
@@ -27,6 +27,34 @@ poetry shell
 
 in which you can run all of the following commands. Alternatively you can prefix all of the following commands with `poetry run`.
 
+### Install PostgreSQL
+
+[Download & Install PostgreSQL](https://www.postgresql.org/download/)
+
+To verify that PostgreSQL is installed on your machine you can use:
+
+```shell
+psql --version
+```
+
+and
+
+```shell
+initdb --version
+```
+
+The latter command (initdb) is required for the application to function correctly.
```

Comment:
this is for running the uni tests as above. You could connect the app to any db, including an externally hosted postgres instance.